docs/guides/integrating-cert-manager.md

```diff
@@ -0,0 +1,325 @@
+
+## Prerequisities
+
+1. Administrator access to a Kubernetes cluster is required.
+2. [Helm](https://helm.sh) and [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) must be installed locally.
+3. Deploy NGINX Kubernetes Gateway (NKG) following the [deployment instructions](/docs/installation.md).
+4. A DNS resolvable domain name is required. It must resolve to the public endpoint of the NKG deployment, and this
+   public endpoint must be an external IP address or alias accessible over the internet. The process here will depend
+   on your DNS provider. This DNS name will need to be resolvable from the Let’s Encrypt servers, which may require
+   that you wait for the record to propagate before it will work.
+
+## Overview
+
+![cert-manager ACME Challenge and certificate management with Gateway API](/docs/images/cert-manager-gateway-workflow.png)
+
+The diagram above shows a simplified representation of the cert-manager ACME Challenge and certificate issuance process
+using Gateway API. Please note that not all of the Kubernetes objects created in this process are represented in
+this diagram.
+
+At a high level, the process looks like this:
+
+1. We deploy cert-manager and create a ClusterIssuer which specifies Let’s Encrypt as our CA and Gateway as our ACME
+   HTTP01 Challenge solver.
+2. We create a Gateway resource for our domain (cafe.example.com) and configure cert-manager integration using an
+   annotation.
+3. This kicks off the certificate issuance process – cert-manager contacts Let’s Encrypt to obtain a certificate, and
+   Let’s Encrypt starts the ACME challenge. As part of this challenge, a temporary HTTPRoute resource is created by
+   cert-manager which directs the traffic through NKG to verify we control the domain name in the certificate request.
+4. Once the domain has been verified, the certificate is issued. Cert-manager stores the keypair in a Kubernetes secret
+   that is referenced by the Gateway resource. As a result, NGINX is configured to terminate HTTPS traffic from clients
+   using this signed keypair.
+5. We deploy our application and our HTTPRoute which defines our routing rules. The routing rules defined configure
+   NGINX to direct requests to https://www.cafe.example.com/coffee to our coffee-app application, and to use the https
+   Listener defined in our Gateway resource.
+6. When the client connects to https://www.cafe.example.com/coffee, the request is routed to the coffee-app application
+   and the communication is secured using the signed keypair contained in the cafe-secret Secret.
+7. The certificate will be automatically renewed when it is close to expiry, the Secret will be updated using the new
+   Certificate, and NKG will dynamically update the keypair on the filesystem used by NGINX for HTTPS termination once
+   the Secret is updated.
+
+## Details
+
+### Step 1 – Deploy cert-manager
+
+The first step is to deploy cert-manager onto the cluster.
```

Comment:
```suggestion
### Step 1 – Deploy `cert-manager`

The first step is to deploy `cert-manager` onto the cluster.
```

This suggestion can be completely ignored: it's just common convention to mark commands/processes differently, but will not detract from the actual text. There's other instances of this I haven't marked for if the suggestion isn't accepted.

Reply:
Because it's the name of a project rather than a process or a command, I think I'll leave it as is. Thanks!